docker-compose.yml

```diff
@@ -12,7 +12,8 @@ services:
     links: 
       - mysql
     volumes: 
-      - ${DOCUMENT_ROOT-./www}:/var/www/html
+      - ${DOCUMENT_ROOT}:/var/www/html
```

Comment:
why did you removed the `./www` here in line 15 ?

Reply:
I'm sorry, I'm a bit confused here about how it works. Inside `sample.env` file we could see `DOCUMENT_ROOT=./www` and here we could see in the red line above `- ${DOCUMENT_ROOT-./www}:/var/www/html`. 

Is it going to get resolved to `./www-./www`?

Reply:
@MrOffline77 can you please let me know about this?

Reply:
@MrOffline77 Sorry for bothering you on this. I got the answer to the question from [here](https://docs.docker.com/compose/compose-file/#variable-substitution)

I'll correct the pull request and will restore the `./www`